### ISSUE TITLE
Timezone and Resolution Spoofing... 

### ISSUE THREAD

**Issue by marzametal** [dillbyrne/random-agent-spoofer#128]
I am over the moon with this product. It allowed me to remove two other addons that I had, since they were dong a bit of everything.

The only concern I have is, it falls short just the same as FireGloves addon did. I cannot for the life of me get the time zone spoofing or the screen resolution spoofing to work! When I go to Panopticlick or other similar websites, it display the original time zone and screen resolution... regardless if I choose Random or specify an entry.

I have to admit, I am using Cyberfox 64bit version... maybe the above two details are restrictive to 32bit?

**Comment by dillbyrne** [comment]
Hello @marzametal 

Cyberfox should not make a difference in this case. It works for me on that site.

Can  you try it on a fresh profile with just RAS installed.
https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles

**Comment by marzametal** [comment]
Hi BOSS!
Thanks for that helpful link. I tried your addon in a new profile, and it works. I must now figure out which other addons are causing the blockage.

**Comment by marzametal** [comment]
G'day BOSS... me again.
After creating a new profile, adding RAS and checking the Panopticlick site after adding my addons 1 by 1... I have come to the conclusion that an addon called "CanvasBlocker" prevents the spoofing of screen resolution and time zone.

Here is the link for the addon...
https://addons.mozilla.org/en-US/firefox/addon/canvasblocker/

**Comment by marzametal** [comment]
When I installed a clean Cyberfox profile, the first thing I did was install RAS. I used the Panopticlick website to test out results. The spoofed time zone and screen resolution showed up perfectly on the site. I added the rest of my addons 1 by 1 and they did not affect the spoofed time zone and screen resolution.

After I added CanvasBlocker, and checked Panopticlick, my "real" time zone and screen resolution showed instead of the spoofed results. I disabled CanvasBlocker, restarted Cyberfox and checked time zone and screen resolution again, and the spoofed results were back.

It tried fresh again, new profile. This time I only installed RAS and CanvasBlocker. Before CB, spoofed time zone and screen resolution showed up on Panopticlick. After CB, spoofed results disappeared, real results displayed.
